lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
```

Comment:
yeah that's what I meant, sorry my fingers don't explain my brain very well :stuck_out_tongue_winking_eye: